packages/pluggableWidgets/slider-web/src/Slider.editorConfig.ts

```diff
@@ -0,0 +1,136 @@
+
+const keysToHideByMinValueType: Record<MinValueTypeEnum, Array<keyof SliderPreviewProps>> = {
+    static: ["minAttribute", "expressionMinimumValue"],
+    dynamic: ["staticMinimumValue", "expressionMinimumValue"],
+    expression: ["staticMinimumValue", "minAttribute"]
+};
+
+const keysToHideByMaxValueType: Record<MaxValueTypeEnum, Array<keyof SliderPreviewProps>> = {
+    static: ["maxAttribute", "expressionMaximumValue"],
+    dynamic: ["staticMaximumValue", "expressionMaximumValue"],
+    expression: ["staticMaximumValue", "maxAttribute"]
+};
+
+const keysToHideBySizeType: Record<StepSizeTypeEnum, Array<keyof SliderPreviewProps>> = {
+    static: ["stepAttribute", "expressionStepSize"],
+    dynamic: ["stepValue", "expressionStepSize"],
+    expression: ["stepValue", "stepAttribute"]
+};
+
+export function getProperties(values: SliderPreviewProps, defaultProperties: Properties): Properties {
+    hidePropertiesIn(defaultProperties, values, keysToHideByMinValueType[values.minValueType]);
+
+    hidePropertiesIn(defaultProperties, values, keysToHideByMaxValueType[values.maxValueType]);
+
+    hidePropertiesIn(defaultProperties, values, keysToHideBySizeType[values.stepSizeType]);
+
+    if (!values.showTooltip) {
+        hidePropertyIn(defaultProperties, values, "tooltip");
+    }
+
+    return defaultProperties;
+}
+
+type CheckFn = (props: SliderPreviewProps) => Problem | undefined;
+
+const minValueCheck: CheckFn = ({ minValueType, expressionMinimumValue, minAttribute }) => {
+    if (minValueType === "expression" && expressionMinimumValue === "") {
+        return {
+            property: "expressionMinimumValue",
+            message: "Minimum value expression is empty"
+        };
+    }
+
+    if (minValueType === "dynamic" && minAttribute === "") {
+        return {
+            property: "minAttribute",
+            message: "Missing attribute for 'Minimum value' property"
+        };
+    }
+};
+
+const maxValueCheck: CheckFn = ({ maxValueType, expressionMaximumValue, maxAttribute }) => {
+    if (maxValueType === "expression" && expressionMaximumValue === "") {
+        return {
+            property: "expressionMaximumValue",
+            message: "Maximum value expression is empty"
+        };
+    }
+
+    if (maxValueType === "dynamic" && maxAttribute === "") {
+        return {
+            property: "maxAttribute",
+            message: "Missing attribute for 'Maximum value' property"
+        };
+    }
+};
+
+const minMaxValueCheck: CheckFn = ({ minValueType, maxValueType, staticMinimumValue, staticMaximumValue }) => {
+    const isPossibleToCheck = minValueType === "static" && maxValueType === "static";
+
+    if (!isPossibleToCheck) {
+        return;
+    }
+
+    if (staticMaximumValue === null || staticMinimumValue === null) {
+        return;
+    }
+
+    if (staticMinimumValue < staticMaximumValue) {
+        return;
+    }
```

Comment:
Nit: You could also combine all of them into one big conditional, I don't think that will be bad for readability:
```
if (minValueType !== "static" || maxValueType !== "static" || staticMaximumValue === null || staticMinimumValue === null || staticMinimumValue < staticMaximumValue) {
    return;
}
```

or if you don't like the numerous number of ORs:

Nit: You could also combine all of them into one big conditional, I don't think that will be bad for readability:
```
if (minValueType !== "static" || maxValueType !== "static" || (staticMaximumValue && staticMinimumValue && staticMinimumValue < staticMaximumValue)) {
    return;
}
```